src/popup/pages/Retip.vue

```diff
@@ -33,149 +36,196 @@
     </div>
 
     <BtnMain
+      class="bottom-btn"
+      extend
       :disabled="!tippingSupported || validationStatus.error || $validator.errors.has('amount')"
       @click="sendTip"
     >
       {{ $t('pages.tipPage.confirm') }}
     </BtnMain>
-    <BtnMain @click="openCallbackOrGoHome(false)">
+    <BtnMain
+      class="bottom-btn"
+      extend
+      @click="openCallbackOrGoHome(false)"
+    >
       {{ $t('pages.tipPage.cancel') }}
     </BtnMain>
 
     <Loader v-if="loading" />
   </div>
 </template>
 
-<script>
-import { mapGetters, mapState } from 'vuex';
+<script lang="ts">
+import {
+  defineComponent,
+  onMounted,
+  ref,
+  watch,
+  computed,
+  Ref,
+} from '@vue/composition-api';
 import { SCHEMA } from '@aeternity/aepp-sdk';
 import { MAGNITUDE, AETERNITY_CONTRACT_ID } from '../utils/constants';
-import { convertToken, watchUntilTruthy } from '../utils';
-import deeplinkApi from '../../mixins/deeplinkApi';
-import maxAmountMixin from '../../mixins/maxAmountMixin';
+import { convertToken, watchUntilTruthy, rxJsObservableToVueState } from '../utils';
+import { useMaxAmount } from '../../composables/maxAmount';
+import { useDeepLinkApi } from '../../composables/deepLinkApi';
```

Comment:
I suggest to import the composables from `../../composables/` at once after adding them to the index.ts file.